src/command/stash.ts

```diff
@@ -0,0 +1,76 @@
+import { git } from "../core";
+
+/**
+ * Stash the current changes.
+ *
+ * @param repositoryPath the repository path to the local Git clone.
+ * @param message an optional stash message.
+ */
+export async function stash(repositoryPath: string, message?: string) {
```

Comment:
From the specification:
_"Calling `git stash` without any arguments is equivalent to `git stash push`"_

I propose wrapping all the functions into a `stash` namespace, and rename this `stash` to `push`. WDYT?

So when I import it, on the use-site, I will have `stash.list()` instead of `list`.